Unable to retrieve meteo data
2017-11-06 15:19:51.496177
INSERT INTO Temp2 VALUE (NULL,NOW(),22.5,45.2,1,25.1,42.1,1,25.0,44.2,1,23.0,44.5,1,0,0,0)
INSERT INTO TempHour VALUE (NULL,NOW(),22.5,45.2,1,25.1,42.1,1,25.0,44.2,1,23.0,44.5,1,0,0,0)
2017-11-06 15:19:51.678183
2017-11-06 15:19:52.679267
2017-11-06 15:19:53.680414
2017-11-06 15:19:54.680556
2017-11-06 15:19:55.681677
2017-11-06 15:19:56.682837
2017-11-06 15:19:57.683973
2017-11-06 15:19:58.685108
2017-11-06 15:19:59.686205
2017-11-06 15:20:00.687350
INSERT INTO Temp2 VALUE (NULL,NOW(),22.5,45.2,1,25.1,41.9,1,25.0,44.0,1,23.0,44.5,1,0,0,0)
2017-11-06 15:20:00.739757
2017-11-06 15:20:01.740850
2017-11-06 15:20:02.741998
2017-11-06 15:20:03.743150
2017-11-06 15:20:04.744284
2017-11-06 15:20:05.745423
2017-11-06 15:20:06.746571
2017-11-06 15:20:07.747730
2017-11-06 15:20:08.748891
2017-11-06 15:20:09.749999
2017-11-06 15:20:10.751102
2017-11-06 15:20:11.752246
2017-11-06 15:20:12.753353
Traceback (most recent call last):
  File "TempObject.py", line 140, in <module>
    time.sleep(1)
KeyboardInterrupt
